pkg/org/org.go

```diff
@@ -57,5 +77,39 @@ func GetEnvironments(ArmoryCloudAddr *url.URL, accessToken *string) ([]Environme
 	if err != nil {
 		return nil, err
 	}
-	return nil, fmt.Errorf("error retrieving environment to login to. ErrorId: %s, Desc: %s", errorResponse.ErrorId, errorResponse.Errors)
+	return nil, fmt.Errorf("error retrieving environment to login to. ErrorId: %s, Desc: %v", errorResponse.ErrorId, errorResponse.Errors)
+}
+
+func GetAgents(ArmoryCloudAddr *url.URL, accessToken string) ([]Agent, error) {
+	connectedAgentsUrl := &url.URL{
+		Scheme: ArmoryCloudAddr.Scheme,
+		Host:   ArmoryCloudAddr.Host,
+		Path:   CONNECTED_AGENTS_URI,
+	}
+	request := util.NewHttpRequest("GET", connectedAgentsUrl.String(), nil, &accessToken)
+	resp, err := request.Execute()
+	if err != nil {
+		return nil, errors.New("Unable to retrieve Remote Network Agents to connect with. Please ensure a Remote Network Agent is connected and try again")
+	}
+	defer resp.Body.Close()
+	dec := json.NewDecoder(resp.Body)
+	if resp.StatusCode == 200 {
+		var connectedAgents []Agent
+		err = dec.Decode(&connectedAgents)
+		if err != nil {
+			return nil, err
+		}
+		return connectedAgents, nil
+	}
+
+	if resp.StatusCode == 401 {
+		return nil, fmt.Errorf("Error: Unauthorized. Run `armory login` to ensure you're using the correct tenant.")
```

Comment:
![image](https://user-images.githubusercontent.com/711726/171349313-9666fd86-b1c4-412d-82b9-ebf0d30f38ef.png)
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

I've been trying to follow this guidance.

Reply:
So you should add the punctuation `.` at the top level where you are actually logging this.

Reply:
I'm not going to repeat this comment on the other `fmt.ErrorF` statements, but it's still applicable to them as well.

Reply:
So, the problem that I have with that guidance is it goes against what the docs team is asking for. Is "because they might appear in another context" a legitimate concern for our use case?